pkg/temporal/ctxpropagation/propagator.go

```diff
@@ -0,0 +1,81 @@
+package ctxpropagation
+
+import (
+	"context"
+	"go.opentelemetry.io/otel/baggage"
+
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/workflow"
+)
+
+type (
+	// contextKey is an unexported type used as key for items stored in the
+	// Context object
+	contextKey struct{}
+
+	// propagator implements the custom context propagator
+	propagator struct{}
+)
+
+type Values baggage.Baggage
```

Comment:
Pulled verbatim from [here](https://github.com/temporalio/samples-go/blob/v1.3.0/ctxpropagation/propagator.go#L19-L22), except with just this line changed